pdfminer/layout.py

```diff
@@ -230,6 +231,7 @@ def __init__(
         non_stroking_color: Optional[Color] = None,
         original_path: Optional[List[PathSegment]] = None,
         dashing_style: Optional[Tuple[object, object]] = None,
+        ocg: str = None
```

Comment:
If the `ocg` is not changing how we draw `LTCurve`'s I rather not store it here. Remember that the `LTCurve` is used by pdfminer.six's layout algorithm to reconstruct the pdf as (for example) an html.

Reply:
Hi Pieter! First off, thanks for reviewing this MR. 

I remember adding this here for the use case of a user wanting to maintain the OCG attribute when reconstructing the PDF for online mapping purposes. 

_As an example:_ Let's say a given PDF represents the map of a small town. LTCurves, LTRects, and LTLines comprise "Rivers", "Houses", "Property Boundaries", etc, each of which is a separate layer on the original PDF. Text on the page describes landmarks, county names, etc. I give this example because smaller-scale PDF map digitalization is often a huge modern inconvenience for environmental/local entities.

If the user wants to extract the LTCurves/Rects/Lines from the PDF to put into an online GIS map, they'll also want the attributed OCGs so that they can make appropriate map layers. With many LTCurves/Rects/Lines often comprising one visible shape, it would be a headache for the user to assign the individual OCGs manually. 

In the `extract_pages` functionality, when using the `if isinstance(element, LTElementName):`, the user can easily get the OCG for the element alongside other pertinent information that already exists for the LTCurve object such as color, path, & dashing style. They can store this information for each LTElement, and then use it when they recreate the map digitally.